orion/controllers/programs/programsController.go

```diff
@@ -9,6 +9,7 @@ import (
 
 const REGEX_PROGRAM_ID = `^[[:alnum:]]+(_[[:alnum:]]+)*$`
```

Comment:
I shouldn't mentioned before. Could we get a comment per program id, name and description?
Basically it would be nice to have a summary of what these REGEX are checking.